mlrun/utils/helpers.py

```diff
@@ -1622,3 +1622,11 @@ def get_local_file_schema() -> List:
     # The expression `list(string.ascii_lowercase)` generates a list of lowercase alphabets,
     # which corresponds to drive letters in Windows file paths such as `C:/Windows/path`.
     return ["file"] + list(string.ascii_lowercase)
+
+
+def resolve_safe_path(base, filepath, is_symlink=False):
```

Comment:
seems more fitting ,also please add type hints
```suggestion
def is_safe_path(base, filepath, is_symlink=False):
```